include/xsimd/types/xsimd_avx512_int32.hpp

```diff
@@ -176,6 +176,45 @@ namespace xsimd
                 return _mm512_sub_epi32(lhs, rhs);
             }
 
+            static batch_type sadd(const batch_type& lhs, const batch_type& rhs)
```

Comment:
I'll benchmark that approach compared to the one based on a comparison + a blend.

Reply:
Actually, found a nice and efficient solution based on min/max \o/